server/eventlog.go

```diff
@@ -127,9 +151,9 @@ func getVerifiedCosState(coscel cel.CEL) (*pb.AttestedCosState, error) {
 
 	seenSeparator := false
 	for _, record := range coscel.Records {
-		// COS State only comes from the CosEventPCR
-		if record.PCR != cel.CosEventPCR {
-			return nil, fmt.Errorf("found unexpected PCR %d in CEL log", record.PCR)
+		// COS State only comes from the CosEventPCR or CosEventLogRegisterIndex
+		if record.PCR != cel.CosEventPCR && record.PCR != cel.CosEventLogRegisterIndex {
```

Comment:
Could you help me underestand why checking `record.PCR` not equal to 4? Looks like we'll call `rtmr.ExtendDigestClient` to measure container state into `cel.CosRTMR` which is 3?

Reply:
In the CCEL, the index that represents RTMR3 is 4.

For configfstsm, the index that represents RTMR3 is 3.

It is very confusing. In fact, Intel also made the similar mistakes (see https://github.com/tianocore/edk2/pull/6051).